.babelrc

```diff
@@ -1,4 +1,4 @@
 {
   "presets": ["es2015", "react", "stage-0"],
-  "plugins": ["add-module-exports"]
+  "plugins": ["add-module-exports", "transform-flow-strip-types"]
```

Comment:
I'm wondering how we should deliver flow types to npm so people can get advantage of them in their projects? @ryyppy I know you did this before.